apps/site/src/_pages/docs/5_faq.mdx

```diff
@@ -209,6 +209,30 @@ We do not yet believe it necessary to specify block-application requests in Grap
 
 </Box>
 
+## Endpoints and APIs
+
+<Box>
+
+<FAQ question="Can blocks access external APIs?">
+
+Yes, blocks can make requests to external APIs. They can do this in one of two ways:
+- **directly:** all of the code require to access an API can be contained within a block. This is not recommended in circumstances where a service requires an API key in order to access it, as it will be impossible to keep this key secret if a block (and for a block to remain usable) once the block is published to the Þ Hub.
+- **through the Þ Hub:** in addition to blocks and types, endpoints (APIs) can also be made accessible via the Þ Hub. A limited set of endpoints will be available as of 28th February 2023, including ones providing access to _Mapbox_ and _OpenAI_.
+  
+Facilitating external API access via the Þ Hub is a new addition to the protocol, and intructions for making additional endpoints available will be added in due course. In the meantime, we're able to manually add endpoints on behalf of block developers. If there's an endpoint you'd like to use within a new or existing block, please [message us on Discord](https://blockprotocol.org/discord).
+
+</FAQ>
+  
+<FAQ question="Can blocks access external APIs while authenticating as users?">
+  
+The ability for Þ blocks to access external services while (optionally) authenticated as an individual user is a planned capability. A technical solution has been identified which will enable this securely and seamlessly across embedding environments, and we intend to introduce support in the coming months. This will enable blocks to be developed that allow for a much wider range of use-cases to be addressed.
```

Comment:
I would feel better about this as I don't think we've fully detailed the solution yet, and it makes the same point
```suggestion
The ability for Þ blocks to access external services while (optionally) authenticated as an individual user is a capability which we intend to introduce in the coming months. This will enable blocks to be developed that allow for a much wider range of use-cases to be addressed.
```